terraform/zones/zone.alexandria.ucsb.edu.tf

```diff
@@ -2,6 +2,18 @@ locals {
   alex-zone_id = aws_route53_zone.r53_zones["alexandria-ucsb-edu"].zone_id
 }
 
+# refer to zone.library.ucsb.edu.tf for aws_lb data definition
+resource "aws_route53_record" "wildcard-eks-dld-alexandria-ucsb-edu-A" {
+zone_id = local.library-zone_id
+  name    = "*.eks.dld.alexandria.ucsb.edu."
```

Comment:
@michael-cr41g i think this wants to either be `*.alexandria.ucsb.edu` or `piru.alexandria.ucsb.edu`, depending on whether there are any other subdomains of `alexandria.ucsb.edu` that need to continue to point somewhere else.

if in doubt, i think this record should just be for `piru.alexandria.ucsb.edu` right?

Reply:
```suggestion
  name    = "piru.alexandria.ucsb.edu."
```